src/Microsoft.AspNet.Mvc.Core/Formatters/JsonOutputFormatter.cs

```diff
@@ -22,6 +22,8 @@ public JsonOutputFormatter()
             SupportedMediaTypes.Add(MediaTypeHeaderValue.Parse("text/json"));
 
             _serializerSettings = new JsonSerializerSettings();
+
+            _serializerSettings.ContractResolver = new JsonContractResolver();
```

Comment:
FYI: With the current state of the contract resolver, it ideally applies only to the input side...but I applied it to output too as we plan to support camel case etc. in future in which case we need this on both input & output...